src/Ursa.Themes.Semi/Controls/TreeComboBox.axaml

```diff
@@ -0,0 +1,293 @@
+                            </MultiBinding>
+                        </ContentPresenter.ContentTemplate>
+                    </ContentPresenter>
+                    <Button
+                        Name="PART_ClearButton"
+                        Grid.Column="2"
+                        Command="{Binding $parent[iri:IClearControl].Clear}"
+                        IsVisible="False"
+                        Theme="{DynamicResource InnerIconButton}"
+                        Content="{DynamicResource IconButtonClearData}" />
+                    <ContentPresenter
+                        Grid.Column="3"
+                        Margin="0 0 8 0"
+                        VerticalAlignment="Center"
+                        VerticalContentAlignment="Center"
+                        Content="{TemplateBinding InnerRightContent}"
+                        Foreground="{DynamicResource TextBoxInnerForeground}"
+                        IsVisible="{TemplateBinding InnerRightContent,
+                                                    Converter={x:Static ObjectConverters.IsNotNull}}" />
+                    <Panel
+                        Grid.Column="4"
+                        Width="32"
+                        Background="Transparent"
+                        IsHitTestVisible="True">
+                        <PathIcon
+                            x:Name="DropDownGlyph"
+                            Width="12"
+                            Height="12"
+                            Margin="0,0,10,0"
+                            HorizontalAlignment="Right"
+                            VerticalAlignment="Center"
+                            Data="{DynamicResource ComboBoxIcon}"
+                            Foreground="{DynamicResource ComboBoxIconDefaultForeground}"
+                            UseLayoutRounding="False" />
+                    </Panel>
+                    <Popup
+                        Name="{x:Static iri:PartNames.PART_Popup}"
+                        Grid.Column="0"
+                        MinWidth="{Binding Bounds.Width, RelativeSource={RelativeSource TemplatedParent}}"
+                        MinHeight="{TemplateBinding MaxDropDownHeight}"
+                        ClipToBounds="False"
+                        InheritsTransform="True"
+                        IsLightDismissEnabled="True"
+                        IsOpen="{Binding IsDropDownOpen, RelativeSource={RelativeSource TemplatedParent}, Mode=TwoWay}"
+                        PlacementTarget="Background"
+                        WindowManagerAddShadowHint="False">
+                        <Border
+                            Name="PopupBorder"
+                            Margin="0,4"
+                            Background="{DynamicResource ComboBoxPopupBackground}"
+                            BorderBrush="{DynamicResource ComboBoxPopupBorderBrush}"
+                            BoxShadow="{DynamicResource ComboBoxPopupBoxShadow}"
```

Comment:
Missing BorderThickness

Reply:
The BoxShadow is not obvious.

Reply:
Added BorderThickness. shadow is also not obvious in combobox, will fix with that together if any idea.